pinot-servicemanager/docker-bin/start-servicemanager

```diff
@@ -5,6 +5,21 @@
 # Dockerfile instructions to WORKDIR
 set -eu
 
+# Apply one-time deferred configuration that relies on ENV variables
```

Comment:
pinot seems to require absolute paths, which is annoying. https://apache-pinot.slack.com/archives/CDRCA57FC/p1599123680104000